composer.json

```diff
@@ -12,12 +12,12 @@
         { "name": "Jeremy Mikola", "email": "jmikola@gmail.com" }
     ],
     "require": {
-        "php": ">=5.3.2",
+        "php": ">=5.4.0",
         "symfony/framework-bundle": "~2.3|~3.0",
         "symfony/console": "~2.1|~3.0",
         "symfony/form": "~2.1|~3.0",
         "symfony/property-access": "~2.3|~3.0",
-        "ruflin/elastica": "~2.1",
+        "ruflin/elastica": "3.0.*",
```

Comment:
`3.0.*` -> `^3.0`.

> Not sure if we want to drop Elastic 1.7 support, ping @merk

Any chance to support both version of `ruflin/elastica` to make it BC?